### PR TITLE
Grammar/spelling improvements for the docs in data.rb.

### PR DESCRIPTION
**Pull request by connorshea**
Also adds some examples for a few methods.

### CODE REVIEW

File: lib/discordrb/data.rb [hunk 1]
```diff
@@ -2585,31 +2589,33 @@ def create_reaction(reaction)
 
     alias_method :react, :create_reaction
 
-    # Returns the list of users who reacted with a certain reaction
+    # Returns the list of users who reacted with a certain reaction.
     # @param reaction [String, #to_reaction] the unicode emoji or {Emoji}
+    # @example Get all the users that reacted with a thumbsup.
+    #   thumbs_up_reactions = message.reacted_with(":thumbsup:")
```
Comment (z64) [inline lib/discordrb/data.rb]: The reaction API does not function with shortcodes

File: lib/discordrb/data.rb [hunk 2]
```diff
@@ -2545,7 +2549,7 @@ def scan_for_emoji
       emoji
     end
 
-    # @return [Array<Emoji>] the emotes that were used/mentioned in this message (Only returns Emoji the bot has access to, else nil).
+    # @return [Array<Emoji>] the emotes that were used/mentioned in this message (Only returns Emoji the bot has access to, else `nil`).
```
Comment (z64) [inline lib/discordrb/data.rb]: I don't think the last part of this docstring is correct anymore (it should be removed)

See https://github.com/meew0/discordrb/pull/473

File: lib/discordrb/data.rb (hunk 1 above)
Comment (connorshea) [reply lib/discordrb/data.rb]: Originally this line was the following:

`thumbs_up_reactions = message.reacted_with("👍")`

But Rubocop didn't like that, can I just disable that cop for this bit of the code?

Comment (z64) [reply lib/discordrb/data.rb]: I think encouraging the use of escapes with the correct codepoints is best: `"\u{1F44D}"`

**Comment by z64** [comment]
Thank you!

**Pull request merged**